# Attitude Dynamics
t_pitch: 0.15
k_pitch: 1.02
t_roll: 0.15
k_roll: 1.02

# Dampening coefficients
damping_x: -0.38
damping_y: -0.38
damping_z: -0.10

# Thrust coefficient
k_thrust: 13.74

# Gravity on world frame
gravity: -9.8066